edx_credentials_themes/locale/en/LC_MESSAGES/django.po

```diff
@@ -16,107 +16,154 @@ msgstr ""
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 
-#: edx_credentials_themes/templates/edx.org/credentials/programs/base.html:18
+#: templates/edx.org/credentials/programs/base.html:19
```

Comment:
I'm not sure why the translations file my virtualenv built is missing the edx_credentials_themes/ part of the path. 
@amangano-edx any idea why/if this will be a problem? (it didnt seem to cause issues when I used this branch in credentials)

Reply:
It's probably because the locale directory is now a child of edx_credentials_themes. I don't think it was when I original generated the files. I don't think it's an issue.